app-shells/atuin/atuin-18.5.0.ebuild

```
@@ -5,21 +5,22 @@ EAPI=8
 
 CRATES=""
 
-RUST_MIN_VER="1.71.1"
+RUST_MIN_VER="1.86"
 
 inherit cargo greadme shell-completion systemd
 
 DESCRIPTION="Shell history manager supporting encrypted synchronisation"
 HOMEPAGE="https://atuin.sh https://github.com/atuinsh/atuin"
 SRC_URI="https://github.com/atuinsh/${PN}/archive/refs/tags/v${PV}.tar.gz -> ${P}.tar.gz"
-SRC_URI+=" https://github.com/gentoo-crate-dist/atuin/releases/download/v${PV}/${P}-crates.tar.xz"
+DEPS_URI="https://github.com/freijon/${PN}/releases/download/v${PV}/${P}-crates.tar.xz"
+SRC_URI+=" ${DEPS_URI}"
```

```suggestion
SRC_URI+=" https://github.com/gentoo-crate-dist/atuin/releases/download/v${PV}/${P}-crates.tar.xz"
```

It was well generated, why use another source?
https://github.com/gentoo-crate-dist/atuin/releases/tag/v18.5.0